native/cocos/math/Vec3.h

```diff
@@ -615,6 +615,10 @@ class CC_DLL Vec3 {
     static const Vec3 UNIT_Z;
     /** equals to Vec3(0,0,-1) */
     static const Vec3 FORWARD;
+
+private:
+    void transformMat4_c(const Vec3 &v, const Mat4 &m);
+    void transformMat4_neon(const Vec3 &v, const Mat4 &m);
```

Comment:
These ethods can be anonymous in Vec3.cpp?

Reply:
No, it's because `MathUtil::transformVec4` is private method and Vec3 is the friend class of MathUtil.